modules/cli/src/main/scala/coursier/cli/bootstrap/GraalvmOptions.scala

```diff
@@ -14,7 +14,7 @@ final case class GraalvmOptions(
     intermediateAssembly: Boolean = false,
   @Group("Graalvm")
   @Hidden
-  @Help("GraalVM version to use to generate native images")
+  @Help("Use a specific GraalVM version to use to generate native images, such as `22.3`, or `graalvm:21`, or `system`")
```

Comment:
NIT I wouldn't change the beginning of the message here:
```suggestion
  @Help("GraalVM version to use to generate native images, such as `22.3`, or `graalvm:21`, or `system`")
```

Or it should be a more edited then, like `Use a specific GraalVM version to generate native images, ...`